content/articles/how-to-create-a-rest-api-using-spring-boot-maven-and-mysql/index.md

```diff
@@ -0,0 +1,810 @@
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
+
+
+```
+
+### 8. Payloads
+
+Inside the data package, create a package with the name payloads. This package will have two sub-packages “request” and “response” to handle our request payloads and response payloads respectively.
+####  8.1 Request Payloads
+Inside the request package create an EmployeeRequest class with the following code:
+```java
+package com.lekwacious.employee_app.data.payloads.request;
+import com.lekwacious.employee_app.data.models.Department;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class EmployeeRequest {
+    @NotBlank
+    @NotNull
+    private String firstName;
+    @NotBlank
+    @NotNull
+    private String lastname;
+    @NotBlank
+    @NotNull
+    private String phoneNumber;
+    @Email
+    private String email;
+    @NotBlank
+    @NotNull
+    private double salary;
+    @NotBlank
+    @NotNull
+    @Enumerated(EnumType.STRING)
+    private Department department;
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastname() {
+        return lastname;
+    }
+
+    public void setLastname(String lastname) {
+        this.lastname = lastname;
+    }
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+    public void setPhoneNumber(String phoneNumber) {
+        this.phoneNumber = phoneNumber;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public double getSalary() {
+        return salary;
+    }
+
+    public void setSalary(double salary) {
+        this.salary = salary;
+    }
+
+    public Department getDepartment() {
+        return department;
+    }
+
+    public void setDepartment(Department department) {
+        this.department = department;
+    }
```

Comment:
You can remove these getter/setter methods. Just explain the change here.

Our goal here is not to type out full codes in a tutorial. You can just type out necessary part (not redundant), and put the rest in a GitHub repo, and link them here.